.circleci/config.yml

```diff
@@ -20,37 +25,36 @@ jobs:
       - run:
           name: Install AWS CLI
           command: |
-            sudo apt-get -y -qq update
-            sudo apt-get -y -qq install awscli
+            sudo apt-get install -y python-pip libyaml-dev python-dev
+            sudo pip install awscli
```

Comment:
why do we need to change this?  That seemed to be working fine before.

Reply:
This will give us the most up to date cli which I need to do the deploy actions & I'd like to be consistent on how we install the cli as a whole